modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/util/v1_0/OrderItemUtil.java

```diff
@@ -90,7 +92,8 @@ public static CommerceOrderItem addCommerceOrderItem(
 
 			commerceOrderItem =
 				commerceOrderItemService.importCommerceOrderItem(
-					orderItem.getExternalReferenceCode(), orderItem.getId(),
+					GetterUtil.getString(orderItem.getExternalReferenceCode()),
+					GetterUtil.getLong(orderItem.getId()),
```

Comment:
@luca-pellizzon I've used GetterUtil, jus in case one of the two is not passed, which in the case of new order item I think none of the two should be passed